packages/terra-application-navigation/src/common/PopupCount.module.scss

```diff
@@ -1,3 +1,7 @@
+// Themes
+@import '../clinical-lowlight-theme/PopupCommon.module';
+@import '../orion-fusion-theme/PopupCommon.module';
```

Comment:
Why are these pulling from a common file instead of PopupCount file?

Reply:
There are about 20 different scss files under popup, drawerMenu, extensions and other subcomponents. So I thought instead of having 40 different theme files with 4-5 variables each its better to have a common theme file for each subcomponent.